test/test_schedule.py

```diff
@@ -40,6 +42,22 @@ def check_schedule(t:Union[Tensor, List[Tensor]], allowed:int, to_prerealize:Opt
     l.linearize()
   return sched
 
+class CycleBitcast(Function):
```

Comment:
I don't think this is a good test. If I run:
```
  def bitwise_cast(self, x: LazyBuffer):
    return x.cast(dtypes.int32, True)
```
instead it also passes with only 1 ScheduleItem.